php/class-fieldmanager-field.php

```diff
@@ -979,8 +983,49 @@ public function add_page_form( $uniqid ) {
 	 * @param int $parent Only show this field on child terms of this parent term ID
 	 */
 	public function add_term_form( $title, $taxonomies, $show_on_add = true, $show_on_edit = true, $parent = '' ) {
+		_deprecated_function( __METHOD__, 'Fieldmanager-1.0.0-beta.3', 'Fieldmanager_Field::add_term_meta_box()' );
```

Comment:
Do we need a WordPress version check here?

If I'm running < WordPress 4.4, I think I'd need to continue to use `add_term_form()`